pages/guestbook.js

```diff
@@ -23,17 +23,15 @@ export default function GuestbookPage({ initialEntries }) {
 }
 
 export async function getStaticProps() {
-  const entries = (await redis.hvals('guestbook'))
-    .map((entry) => {
-      const { email, ...restOfEntry } = JSON.parse(entry);
-
-      return restOfEntry;
-    })
-    .sort((a, b) => b.id - a.id);
+  // this needs to be able to be serialized
+  // const [rows] = await db.query(`
+  //   SELECT id, body, created_by FROM guestbook
+  //   ORDER BY id;
+  // `);
```

Comment:
Database column values typically need to be mapped into a type that can be serialized into JSON before returning as a prop. I'm not sure which of these three fields is not serializable, but here's an example mapping that converts `body` to a String.

```js
const entries = rows.map(({id, body, created_by}) => ({id: id, body: String(body), created_by: created_by}))
```

Reply:
Thank you!